test/com/amazon/ion/LoaderTest.java

```diff
@@ -160,6 +160,16 @@ public void testIgnoreHeaderSymbol()
         checkInt(123, value);
     }
 
+    @Test
+    public void testLargeSidSymbol()
+    {
+        String text = "$11111111111111";
+
+        // it's supposed to throw an IonException since sid is greater than INT_MAX.
+        try {
+            IonSymbol value = (IonSymbol) loadOneValue(text);
```

Comment:
You need to add
```
Assert.fail("Expected IonException to be thrown.")
```
after this line. Otherwise, the test will succeed even if the exception is not thrown.

Reply:
Got it, added.